portfolio/src/main/webapp/community.js

```diff
@@ -1,57 +1,58 @@
-    while (commentSection.lastChild) {
-        commentSection.removeChild(commentSection.lastChild);
-    }
-
-    let commentCount = 0;
-    conversation.forEach((comment) => {
-        createComment(comment, commentCount%2);
-        commentCount++;
-    });
-  });
+  let commentRequestedNum =
+      document.getElementById('commentRequestedNum').value;
+  fetch('/comments?commentRequestedNum=' + commentRequestedNum)
+      .then(response => response.json())
+      .then((conversation) => {
+        let commentSection = document.getElementById('submitted-comments');
+        while (commentSection.lastChild) {
+          commentSection.removeChild(commentSection.lastChild);
+        }
+
+        let commentCount = 0;
+        conversation.forEach((comment) => {
+          createComment(comment, commentCount % 2);
+          commentCount++;
+        });
+      });
 }
 
 function createComment(comment, color) {
-    const commentSection = document.getElementById('submitted-comments');
-    const commentDivElement = document.createElement('div');
-    const usernameElement = document.createElement('p');
-    const commentElement = document.createElement('p');
-    const exitButtonElement = document.createElement('button');
+  const commentSection = document.getElementById('submitted-comments');
+  const commentDivElement = document.createElement('div');
+  const usernameElement = document.createElement('p');
+  const commentElement = document.createElement('p');
+  const exitButtonElement = document.createElement('button');
+
+  let commentDivClasses = new Array(2);
+  commentDivClasses[0] = 'comment-div';
 
-    let commentDivClasses = new Array(2);
-    commentDivClasses[0] = 'comment-div';
+  exitButtonElement.className = 'left';
 
-    exitButtonElement.className = 'left';
+  commentDivClasses[1] = color===0 ? 'lavender' : "lavender-blush";
 
-    if (color===0) {
-        commentDivClasses[1] = "lavender";
-    } else {
-        commentDivClasses[1] = "lavender-blush";
-    }
+  commentDivElement.classList.add(...commentDivClasses);
 
-    console.log(commentDivClasses);
+  let usernameText = document.createTextNode(comment.username + ' says: ');
+  usernameElement.appendChild(usernameText);
 
-    commentDivElement.classList.add(...commentDivClasses);
+  let commentText = document.createTextNode(comment.commentText);
+  commentElement.appendChild(commentText);
 
-    usernameElement.innerHTML = comment.username + " says:";
-    commentElement.innerHTML = comment.commentText;
-    exitButtonElement.innerHTML = "×";
+  let exitText = document.createTextNode('×');
+  exitButtonElement.appendChild(exitText);
 
-    exitButtonElement.addEventListener('click', () => {
-        const params = new URLSearchParams;
-        params.append('id', comment.id);
-        fetch('/delete-comment', {method: "POST", body:params});
+  exitButtonElement.addEventListener('click', () => {
+    const params = new URLSearchParams;
```

Comment:
Even though you can technically omit the parentheses when calling a constructor with no arguments, the Google style guide asks us to include the parentheses anyway.

https://google.github.io/styleguide/jsguide.html#disallowed-features-omitting-parents-with-new

Reply:
Added parens